modules/mono/glue/cs_files/Mathf.cs

```diff
@@ -257,5 +257,17 @@ public static real_t Tanh(real_t s)
         {
             return (real_t)Math.Tanh(s);
         }
+
+        public static int Wrap(int val, int min, int max)
+        {
+            int rng = max - min;
+            return min + ((((val - min) % rng) + rng) % rng);
+        }
+
+        public static real_t Wrap(real_t val, real_t min, real_t max)
+        {
+            real_t rng = max - min;
+            return min + (val - min) - (rng * Floor((val - min) / rng));
```

Comment:
`min + (val - min)`? That's just equal to `val` isn't it? `return val - (rng * Floor((val - min) / rng));` gives me equal results.

What is the purpose of being different from the `int` version? `return min + ((((val - min) % rng) + rng) % rng);` seems to work fine here too. Does it not work properly in some cases?

Reply:
Heh. You right, it could be changed to simple `return min + ((((val - min) % rng) + rng) % rng);` I didnt knew that they implement % on floats in C#..

Reply:
In C++ using modulo operator on floating point numbers is impossible, but I changed it for C# in #18038

Reply:
About `return val - (rng * Floor((val - min) / rng));`, you also right, and I will change it soon(for GDScript)..

Reply:
I fixed it in #18170